app/src/main/res/values-v21/styles.xml

```diff
@@ -2,14 +2,7 @@
 <resources>
 
     <style name="AppTheme" parent="AppTheme.Base">
-        <!-- remove action bar, using toolbar -->
-        <item name="windowActionBar">false</item>
-        <item name="android:windowNoTitle">true</item>
-
-        <!-- colors -->
-        <item name="android:colorPrimary">@color/primary</item>
-        <item name="android:colorPrimaryDark">@color/primary_dark</item>
-        <item name="android:colorAccent">@color/accent</item>
+
```

Comment:
did you change application theme in this commit? what are primary and accent colours now?